CHANGELOG.md

```diff
@@ -1,7 +1,12 @@
+1.0.1 (2023-02-15)
+==================
+
+- [chore] [#30](https://github.com/lulichn/embulk-input-dynamodb/pull/30) Upgrade dependencies.
```

Comment:
```suggestion
- [chore] [#29](https://github.com/lulichn/embulk-input-dynamodb/pull/29) Upgrade dependencies.
    - Use `Exec.getPageBuilder` instead of `new PageBuilder`.
    - Upgrade scala 2.13.1 -> 2.13.10
    - Upgrade aws-sdk 1.11.171 -> 1.12.406
```